### PR TITLE
Feature/setup env kubernetes

### PR DESCRIPTION
**Pull request by TiagoMF20**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,9 +155,9 @@ Distributed under the Apache-2.0 License. See `LICENSE` for more information.
 
 ## Contact
 
-Your Name - [@twitter_handle](https://twitter.com/twitter_handle) - email
+Tiago Mendonça Fernandes - tiago.fernandes@elecnor.es
```
Comment (AngelLozanoDeimos) [inline README.md]: Maybe we should replace the email with the team-generic one: eoepca.systemteam@telespazio.com

File: src/nginx/nginx.yaml [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    nginx.ingress.kubernetes.io/session-cookie-name: "route"
+    nginx.ingress.kubernetes.io/session-cookie-hash: "sha1"
+    nginx.ingress.kubernetes.io/proxy-next-upstream: "error timeout invalid_header http_500 http_502 http_503 http_504"
+spec:
+  tls:
+    - hosts:
+      - eoepca-dev.gluu.org
+      secretName: tls-certificate
+  rules:
+  - host: eoepca-dev.gluu.org
```
Comment (AngelLozanoDeimos) [inline src/nginx/nginx.yaml]: If at all possible, we should make any host/dns configurable, as this is the same code that will be executed in production, or even other platforms, and we can't be sure of the hostname

File: src/config/generate-config.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  GLUU_SECRET_ADAPTER: "kubernetes"
+  ADMIN_PW: "admin_Abcd1234#"
+  EMAIL: "support@gluu.org"
+  DOMAIN: "eoepca-dev.gluu.org"
+  ORG_NAME: "Deimos"
+  COUNTRY_CODE: "PT"
+  STATE: "NA"
+  CITY: "Lisbon"
+  GLUU_CONFIG_ADAPTER: "kubernetes"
+  LDAP_TYPE: "opendj"
```
Comment (AngelLozanoDeimos) [inline src/config/generate-config.yaml]: Although I understand these are just the "dummy" credentials, we should have a way of giving them values for a real production platform. Environment variables maybe? Or maybe document the location so an operator can edit it

**Review by AngelLozanoDeimos — CHANGES_REQUESTED** [review]
Other than the minor changes requested, looks good to me :)

I haven't had the time to execute the code myself, but I trust Travis did the needful.

File: README.md (hunk 1 above)
Comment (TiagoMF20) [reply README.md]: Changed.

File: src/nginx/nginx.yaml (hunk 2 above)
Comment (TiagoMF20) [reply src/nginx/nginx.yaml]: Changed this to a placeholder variable, that is replaced at runtime in setupCluster.sh. Variable is defined in the script.

File: src/config/generate-config.yaml (hunk 3 above)
Comment (TiagoMF20) [reply src/config/generate-config.yaml]: Moved variables to the setupCluster, and changed these values to placeholder variables. When installing, setupCluster changes these with sed to the real values to perform the installation, but the real values are not stored in these configuration files, only in the setupCluster.sh script.
Changed README to indicate where the values can be edited.

**Pull request merged**